action.yml

```diff
@@ -93,6 +93,17 @@ inputs:
     description: 'Colorize output. Choices are: always, never, raw, auto'
     required: false
     default: 'auto'
+  manual-plugins:
```

Comment:
Maybe we should name this config key `plugins`?

I'm not sure we can call them _manual plugins_. AFAIK, They are plugins that don't have an install script, we need to manually install them.